embulk-standards/src/main/java/org/embulk/standards/CsvParserPlugin.java

```diff
@@ -261,7 +260,7 @@ public void run(TaskSource taskSource, final Schema schema,
                             public void booleanColumn(Column column)
                             {
                                 String v = nextColumn();
-                                if (v == null) {
+                                if (v == null || v.isEmpty()) {
```

Comment:
Previously, an empty string for boolean, long, double, timestamp, json columns behaved as bellow with `null_string: "NULL"`.

1. assign null for a non-quoted empty string, and raises an error for a quoted empty string.

This PR makes empty strings for both a non-quoted string and a quoted empty string with `null_string: "NULL"` for any kinds of columns once. We keep the empty string for string columns, but we have to treat correctly for boolean, long, double, timestamp, and json columns. I think there are two possible specifications.

1. raises an error for both a non-quoted empty string and a quoted  empty string
2. assign null for both a non-quoted empty string and a quoted empty string

I chose the 2nd.